docs/cli/index.md

```diff
@@ -118,44 +118,37 @@ Ouput:
 
 ### Import labels to your project
 
-To import labels, provide a list of files or folders (you can also procide a csv file external_id and label's paths).
+To import labels, provide a list of files or folders.
 
 Label's files are json with the json_response to upload
```

Comment:
may be a good time to fix this sentence:
Label files are JSON files containing labels in the Kili format: the value corresponding to the `jsonResponse` field of a label (see for example: https://docs.kili-technology.com/reference/json-classification-jobs).

Reply:
Agree!